lib/hpack/huffman.ex

```diff
@@ -14,7 +14,7 @@ defmodule HPack.Huffman do
       "hello"
 
   """
-  @spec decode(binary) :: String.t
+  @spec decode(String.t) :: String.t
```

Comment:
This is a binary type, not a String, while a String is always a binary not every binary is a String.